blogs.html

```diff
@@ -40,6 +40,19 @@
         <div class="page-header">
             <h1 id="timeline">Add Links To Your Blog</h1>
         </div>
+        <ul class="timelineBlog">
+            <li class="timelineBlog-inverted">
+                <div class="timelineBlog-badge info"><i class="glyphicon glyphicon-list-alt"></i></div>
+                <div class="timelineBlog-panel">
+                    <div class="timelineBlog-heading">
+                        <h4 class="timelineBlog-title">Ahmed Monjurul Hasan <a href="https://medium.com/@hasan.ahmedmonjurul/getting-started-with-git-in-bertelsmann-tech-scholarship-b5d4b349ac6"><i class="glyphicon glyphicon-link"></i></a></h4>
+                    </div>
+                    <div class="timelineBlog-body">
+                        <p>My blog post gives a hands-on guide on getting started with Git in Bertelsmann Tech Scholarship</p>
+                        <p><small class="text-muted pull-right">Added On 22/12/2019</small></p>
+                    </div>
+                </div>
+            </li>
         <ul class="timelineBlog">
```

Comment:
Hi Ahmed, 

Thank you for the commit. Unfortunately, the <ul class="timelineBlog"> now has two opening tags when it only needs one. Please remove the this opening tag and then commit the changes. 

Many thanks.